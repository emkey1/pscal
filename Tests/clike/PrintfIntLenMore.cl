int main() {
  // Print integers that exercise larger/smaller ranges (format width not asserted here).
  printf("%d %d\n", 1234, 23);
  printf("%d\n", -123456789);
  printf("%d\n", 255);
  printf("%d %d\n", 44, -5);
  return 0;
}
