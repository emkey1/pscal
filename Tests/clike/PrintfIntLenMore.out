1234 23
-123456789
255
44 -5

